56.md

```diff
@@ -48,35 +53,79 @@ Example events
-    [ "e", <eventId>, "illegal"],
-    [ "p", <pubkey>]
+    [ "p", <pubkey>, "nsfw", "", "-"],
+    [ "p", <pubkey>, "sfw"]
+  ],
+  "content": "It's a cute dog!",
+  ...
+}
+
+{
+  "kind": 1984,
+  "tags": [
+    ["e", <event id>, "spam"],
+    ["p", <pubkey>]
   ],
-  "content": "He's insulting the king!",
+  "content": "Spam post",
   ...
 }
 
 {
   "kind": 1984,
   "tags": [
-    [ "p", <impersonator pubkey>, "impersonation"],
-    [ "p", <victim pubkey>]
+    ["e", <event id>, "spam", "topic"],
+    ["p", <pubkey>]
   ],
-  "content": "Profile is imitating #[1]",
+  "content": "Discusses spam",
   ...
 }
 ```
 
 Client behavior
 ---------------
 
-Clients can use reports from friends to make moderation decisions if they
-choose to. For instance, if 3+ of your friends report a profile as explicit,
-clients can have an option to automatically blur photos from said account.
+Labels can be used the same way as hashtags, they can be used for filtering,
+searching, or displayed to draw attention to something.
 
+Clients should have a threshold for labels to be considered legitimate,
+one example is to weigh a label's legitimacy based on how far the labeler
+is from someone you're following.
+
+Clients may, depending on the use case, display the ratio of
+positive versus negative labels, or even use this ratio to dynamically adjust
+the threshold for legitimacy, creating a form of community-driven moderation.  
+Keep in mind that sometimes clients will completely ignore negative labels.
+
+Labels can be used for sensitive content, for instance, if 3+ events
+from an account are labeled as explicit, clients can have an option
+to automatically blur photos from said account.
 
 Relay behavior
 --------------
 
-It is not recommended that relays perform automatic moderation using reports,
-as they can be easily gamed. Admins could use reports from trusted moderators to
+It is not recommended that relays perform automatic moderation using labels,
+as they can be easily gamed. Admins could use labels from trusted moderators to
 takedown illegal or explicit content if the relay does not allow such things.
+
+Reporting (DEPRECATED)
+----------------------
+
+A report is a special label for moderation.  
+A report MUST include a `p` tag referencing the pubkey of the user you
+are reporting.
+
+If reporting a note, an `e` tag MUST also be included referencing the note id.
+
+A `report type` string MUST be included as the 3rd entry to the `e` or `p` tag
+being reported, which consists of the following report types:
+
+- `nudity` - depictions of nudity, porn, etc.
+- `profanity` - profanity, hateful speech, etc.
+- `illegal` - something which may be illegal in some jurisdiction
+- `spam` - spam
+- `impersonation` - someone pretending to be someone else
+
+Some report tags only make sense for profile reports, such as `impersonation`
+
+> Deprecated due to its built-in negativity, emphasizing moderation needs over
+  neutral content classification.
```

Comment:
This section can be removed since we are maintaining backwards compatibility.

Reply:
Good point